### PR TITLE
rpmspec: add --querytags example

### PR DESCRIPTION
**Pull request by mfrw**
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/rpmspec.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `rpmspec --query {{path/to/rpm.spec}}`
 
+- List all queryformat options:
```
Comment (navarroaxel) [inline pages/linux/rpmspec.md:10]: Perhaps this better
```suggestion
- List all `--queryformat` options:
```

Comment (marchersimon) [reply pages/linux/rpmspec.md:10]: Or maybe
```suggestion
- List all options for `--queryformat`:
```

Comment (mfrw) [reply pages/linux/rpmspec.md]: Thank you for the review. This version looks better :)

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @mfrw

**Pull request merged**